### ISSUE TITLE
New tab does not stay on the same profile

### ISSUE THREAD

**Issue by ReporterX** [hultmann/multifox#194]
When your tab is at "Profile ABC" and you open new tab there (either via Ctrl+T or from "New Tab" button), it changes back to the default profile (instead of "Profile ABC").

For example, 
Tab 1 - Profile ABC

(You open a new tab at Tab 1. What it happens)
Tab 1 - Profile ABC
**Tab 2 - Default Profile**

(What I expect)
Tab 1 - Profile ABC
**Tab 2 - Profile ABC**

Please fix it. Thank you. :-)

**Comment by ReporterX** [comment]
Perhaps you can add a new button or menu item called "**Open New Tab in an Identity Profile**"

**Comment by hultmann** [comment]
See https://github.com/hultmann/multifox/issues/199#issuecomment-81264348

**Comment by jetang** [comment]
I believe new tab stayed on the same profile (either via Ctrl+T or triggered after a mouse click on hyper links) some time ago. It was broken recently. Is it intentional?

**Comment by hultmann** [comment]
Nope, Multifox 3.2.1 fixed it. (ctrl+T always resets to default profile, it is expected)
